clients/dataset/dataset_test.go

```diff
@@ -4,20 +4,22 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
-	"github.com/ONSdigital/go-ns/common/commontest"
+	"github.com/ONSdigital/dp-rchttp"
+	"github.com/ONSdigital/go-ns/common"
 	"github.com/pkg/errors"
 	. "github.com/smartystreets/goconvey/convey"
 	"io/ioutil"
 	"net/http"
 	"testing"
-	"github.com/ONSdigital/go-ns/common"
 )
 
 var ctx = context.Background()
 
+const authToken = "auth token"
```

Comment:
You are going to hate me for asking this but...

We should probably assert that header is added to the request in the test....